### PR TITLE
#164798167 Enable users to view and edit their profiles

### PR DESCRIPTION
**Pull request by ezrogha**
#### What does this PR do?
Enable users to view and edit their profiles
#### Description of Task to be completed?
- Enable user to view profile
- Enable user to edit profile
- Enable user to upload their image

#### How should this be manually tested?
- Fetch this branch git fetch origin `ft-user-should-view-and-edit-their-profiles-164798167`
- Checkout to the branch git checkout `ft-user-should-view-and-edit-their-profiles-164798167`
- Run npm start to view the landing page
- Login to the app on `/Login`
- View profile on `/profile`
- Edit profile on `/editprofile`

#### What are the relevant pivotal tracker stories?
[#164798167]( https://www.pivotaltracker.com/story/show/164798167)

#### Screenshots (if appropriate)
Screenshot 2019-05-16 at 23 27 40
Screenshot 2019-05-16 at 23 27 54

### CODE REVIEW

File: src/components/EditProfileInput/profileInput.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.profileInputContainer {
+  margin-bottom: 15px;
+  .profileInput {
+      font-size: 15px;
+      padding: 5px;
```
Comment (dannylwe) [inline src/components/EditProfileInput/profileInput.scss]: @ezrogha kindly convert these to rem

File: src/components/NavBarAccountSection/NavBarAccountSection.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.logoutButton, .dropdown-item {
+  &:active {
+		outline: 1px solid transparent;
+		background-color: #ccc;
+	}
+}
```
Comment (dannylwe) [inline src/components/NavBarAccountSection/NavBarAccountSection.scss]: @ezrogha kindly change px to rem. Also add a new line to bottom of the page

File: src/components/ProgressBar/ProgressBar.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.progressBar {
+    position: relative;
+    height: 3px;
+    margin-bottom: 10px;
```
Comment (dannylwe) [inline src/components/ProgressBar/ProgressBar.scss]: @ezrogha kindly convert px to rem. Also, add a new line to the bottom of the page

File: src/components/ProgressBar/ProgressBar.scss [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    border-radius: 50px;
+    border: 1px solid #172439;
+}
+
+.filler {
+    background-color: #172439;
+    height: 100%;
+    border-radius: inherit;
+    transition: width .2 ease-in;
+}
```
Comment (dannylwe) [inline src/components/ProgressBar/ProgressBar.scss]: @ezrogha kindly add a new line to this file as well

File: src/pages/EditProfile/EditProfile.scss [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+.form {
+	margin-left: 20px;
+	outline: 1px solid red;
+}
+
+#bio {
+	font-size: 15px;
+	padding: 5px;
+	border: 1px solid #283560;
```
Comment (dannylwe) [inline src/pages/EditProfile/EditProfile.scss]: @ezrogha convert px to rem in this file

File: src/pages/ViewProflle/ViewProfile.scss [hunk 6]
```diff
@@ -0,0 +1,69 @@
+.username {
+  font-size: 50px;
+  margin-top: 0px;
```
Comment (dannylwe) [inline src/pages/ViewProflle/ViewProfile.scss]: @ezrogha kindly convert px to rem in this file

File: src/pages/ViewProflle/ViewProfile.scss [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+	overflow: hidden;
+	.image {
+		width: 100%;
+		margin-bottom: 30px;
+	}
+}
+
+.editProfileButtonWrapper {
+	margin-top: 20px;
+}
```
Comment (dannylwe) [inline src/pages/ViewProflle/ViewProfile.scss:76]: @ezrogha add a new line here

**Review by dannylwe — DISMISSED** [review]

File: src/components/EditProfileInput/profileInput.scss (hunk 1 above)
Comment (ezrogha) [reply src/components/EditProfileInput/profileInput.scss]: Thanks for you feedback @dannylwe,
I has been worked upon

File: src/components/NavBarAccountSection/NavBarAccountSection.scss (hunk 2 above)
Comment (ezrogha) [reply src/components/NavBarAccountSection/NavBarAccountSection.scss]: Thanks for you feedback @dannylwe,
I has been worked upon

File: src/pages/PasswordReset/resetPassword.scss [hunk 8]
```diff
@@ -1,8 +1,8 @@
 @import '../../assets/scss/shared.scss';
 
-body {
-  text-align: center;
-}
+// body {
```
Comment (hadijahkyampeire) [inline src/pages/PasswordReset/resetPassword.scss:3]: You should clean up commented code

**Review by hadijahkyampeire — APPROVED** [review]
Generally the work is good, thanks @ezrogha

**Pull request merged**